Packs/SymantecEmailSecurity/Integrations/SymantecEmailSecurity/README.md

```diff
@@ -0,0 +1,1207 @@
+| before | A time stamp value used to select only SUDULS items that were created before this time. Accepted formats: any substring of yyyy-mm-ddThh:mm:ssZ, epoch 1720617001, relative 1 day 2h 3 minute. | Optional |
+| filter_type | A string used to filter emails based on the quarantine type. By default includes the emails quarantined for all types. COMPLIANCE: Includes Content control, DLP and Image control emails. DLP: Includes only DLP emails. Possible values are: SPAM, NEWSLETTER, CI, CO, II, IO, COMPLIANCE, DLP. | Optional |
+| include_deleted | Specifies whether to include items marked as deleted in the search results. Possible values are: true, false. | Optional |
+| user_email | Return only the quarantined emails of the user whose email address is specified. Note: Can only be used by an administrator user. | Optional |
+| admin_domain | Returns the emails quarantined for users in a particular domain. If this parameter is present and has a valid domain name, then items from only that domain are returned. If it has a value of `ALL`, then all domains administered by the user are searched and emails quarantined for users in those domains are returned. Note: Can only be used by an administrator user. | Optional |
+| limit | The maximum number of records to return. Default is 50. | Optional |
+| page | The page number to retrieve records from. | Optional |
+| page_size | The maximum number of records to return per page. Default is 50. Max is 10,000. | Optional |
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| SymantecEmailSecurity.QuarantineEmail.id | String | ID of the item. |
+| SymantecEmailSecurity.QuarantineEmail.metadata.email_date_received | Date | Date the email was received. |
+| SymantecEmailSecurity.QuarantineEmail.metadata.quarantine_info.direction | String | Direction of the email. |
+| SymantecEmailSecurity.QuarantineEmail.metadata.quarantine_info.quarantine_type | String | Quarantine type of the email, can be one of: SPAM, NEWSLETTER, CI, CO, II, IO, COMPLIANCE, DLP. |
+| SymantecEmailSecurity.QuarantineEmail.metadata.email_is_viewed | Bool | Whether the email was viewed. |
+| SymantecEmailSecurity.QuarantineEmail.metadata.email_is_released | Bool | Whether the email was released. |
+| SymantecEmailSecurity.QuarantineEmail.metadata.quarantine_reason | String | Reason why the email was quarantined. |
+| SymantecEmailSecurity.QuarantineEmail.metadata.email_sender | String | Sender of the email. |
+| SymantecEmailSecurity.QuarantineEmail.metadata.service_type | String | Service type used for the email. |
+| SymantecEmailSecurity.QuarantineEmail.metadata.master_recipient | String | Recipient of the email. |
+| SymantecEmailSecurity.QuarantineEmail.metadata.user_id | Number | ID of the user. |
+| SymantecEmailSecurity.QuarantineEmail.metadata.email_envelope_sender | String | Address to respond in the case of bounce messages or errors. |
```

Comment:
```suggestion
| SymantecEmailSecurity.QuarantineEmail.metadata.email_envelope_sender | String | Address to respond to in case of bounce messages or errors. |
```